tools/mo/openvino/tools/mo/main.py

```diff
@@ -388,7 +388,7 @@ def prepare_ir(argv : argparse.Namespace):
                 for extension in argv.extensions.split(','):
                     moc_front_end.add_extension(extension)
             ngraph_function = moc_pipeline(argv, moc_front_end)
-            return graph, ngraph_function
+            return graph, ngraph_function, moc_front_end
```

Comment:
Just a workaround to extend `frontend` lifetime which resolves the problem with segfault (but it is rather not a target solution)

Reply:
In general, Model should hold frontend:
https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/core/model.hpp#L42